docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -1128,10 +1130,12 @@ generally has the form `[[SOURCE-VOLUME|HOST-DIR:]CONTAINER-DIR[:OPTIONS]]`.
 
 If `SOURCE-VOLUME` starts with `.`, Quadlet resolves the path relative to the location of the unit file.
 
-As a special case, if `SOURCE-VOLUME` ends with `.volume`, Quadlet will look for the corresponding `.volume` Quadlet unit.
-If found, Quadlet will use the name of the Volume set in the Unit, otherwise, `systemd-$name` is used.
+Special case:
+
+* If `SOURCE-VOLUME` ends with `.volume`, Quadlet will look for the corresponding `.volume` Quadlet unit. If found, Quadlet will use the name of the Volume set in the Unit, otherwise, `systemd-$name` is used.
+
 The generated systemd service contains a dependency on the service unit generated for that `.volume` unit,
-or on `$name-volume.service` if the `.volume` unit is not found
+or on `$name-volume.service` if the `.volume` unit is not found.
```

Comment:
Again, the `.volume` file must exist. Quadlet will use the name of the volume generated for the `.volume` unit